### PR TITLE
Fix/jw analytics quantile reporting

### PR DESCRIPTION
**Pull request by Pretzi**
## Description

[Monday ticket](add your monday ticket url here)

`If merged this will add the changes in the jwpltx file from the PR: https://github.com/jwplayer/ott-web-app/pull/256 
 the changes were tested in the TBN app and the quantile events are not being send when seeking and the ads events (impressions) are being sent to the correct endpoint
`

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

File: plugins/zapp-analytics-plugin-jw-player/src/jwpltx.ts [hunk 1]
```diff
@@ -7,97 +7,110 @@ export var jwpltx = jwpltx || {};
...
     uri = JSON.parse(JSON.stringify(URI));
     uri.aid = aid;
     uri.bun = bun;
-    uri.fed = fed || "-1";
+    uri.fed = fed;
     uri.id = id;
     uri.t = t;
-    uri.oosv = oosv;
-    uri.av = av;
+
```
Comment (inckie) [inline plugins/zapp-analytics-plugin-jw-player/src/jwpltx.ts:111]: We probably still need these additional params (oosv, av, oaid, ppid), at least some of them were added by reqest

File: plugins/zapp-analytics-plugin-jw-player/src/jwpltx.ts [hunk 2]
```diff
@@ -7,97 +7,110 @@ export var jwpltx = jwpltx || {};
 
 (function (o: JW_OBJ) {
   // Hostname for sending analytics
-  const URL = "https://ihe.jwpltx.com/v1/jwplayer6/ping.gif?";
+  const URL = "https://ihe.jwpltx.com/v1/clienta/ping.gif?";
```
Comment (ranmeirman) [inline plugins/zapp-analytics-plugin-jw-player/src/jwpltx.ts:10]: You should only change this on ads.

**Review by ranmeirman — CHANGES_REQUESTED** [review]

File: plugins/zapp-analytics-plugin-jw-player/src/jwpltx.ts (hunk 2 above)
Comment (Pretzi) [reply plugins/zapp-analytics-plugin-jw-player/src/jwpltx.ts]: Fixed, the code from the PR  https://github.com/jwplayer/ott-web-app/pull/256 already handles this, and I verified it works as expected in the TBN app

File: plugins/zapp-analytics-plugin-jw-player/src/jwpltx.ts (hunk 1 above)
Comment (Pretzi) [reply plugins/zapp-analytics-plugin-jw-player/src/jwpltx.ts]: Fixed

File: plugins/zapp-analytics-plugin-jw-player/src/helper.ts [hunk 3]
```diff
@@ -44,29 +44,16 @@ export enum DATA_EVENTS {
...
   return platformSelect({
-    amazon: JW_PLATFORM.FireOS,
-    android_tv: JW_PLATFORM.AndroidTV,
-    android: JW_PLATFORM.Android,
-    ios: JW_PLATFORM.iOS,
-    lg_tv: JW_PLATFORM["Web OS"],
-    samsung_tv: JW_PLATFORM["Tizen OS"],
-    tvos: JW_PLATFORM.tvOS,
-    web: JW_PLATFORM.Web,
+    amazon: [6, "FireOS"],
```
Comment (inckie) [inline plugins/zapp-analytics-plugin-jw-player/src/helper.ts:49]: Fire TV

File: plugins/zapp-analytics-plugin-jw-player/src/helper.ts [hunk 4]
```diff
@@ -44,29 +44,16 @@ export enum DATA_EVENTS {
...
+    android_tv: [9, "Android TV"],
+    android: [1, "Android"],
+    ios: [2, "iOS"],
+    lg_tv: [8, "Web OS"],
```
Comment (inckie) [inline plugins/zapp-analytics-plugin-jw-player/src/helper.ts:53]: Lets ask them about that one, its not in the allowed list

File: plugins/zapp-analytics-plugin-jw-player/src/helper.ts [hunk 5]
```diff
@@ -44,29 +44,16 @@ export enum DATA_EVENTS {
...
+    samsung_tv: [7, "Tizen OS"],
```
Comment (inckie) [inline plugins/zapp-analytics-plugin-jw-player/src/helper.ts:54]: Doc says its just 'Tizen'

File: plugins/zapp-analytics-plugin-jw-player/src/helper.ts [hunk 6]
```diff
@@ -44,29 +44,16 @@ export enum DATA_EVENTS {
...
-    android: JW_PLATFORM.Android,
-    ios: JW_PLATFORM.iOS,
-    lg_tv: JW_PLATFORM["Web OS"],
-    samsung_tv: JW_PLATFORM["Tizen OS"],
-    tvos: JW_PLATFORM.tvOS,
-    web: JW_PLATFORM.Web,
+    amazon: [6, "FireOS"],
+    android_tv: [9, "Android TV"],
+    android: [1, "Android"],
+    ios: [2, "iOS"],
```
Comment (inckie) [inline plugins/zapp-analytics-plugin-jw-player/src/helper.ts:52]: Lets ask them if they want to have iPadOS on tablets (its in the list)

**Review by inckie — APPROVED** [review]

**Review by ranmeirman — APPROVED** [review]

**Pull request merged**